### PR TITLE
openvpn3: add Chinese translation

### PR DESCRIPTION
**Pull request by wandersofb**
- [x] The page (if new), does not already exist in the repository.
- [x] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [x] The page has 8 or fewer examples.
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- [x] The page follows the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The page description includes a link to documentation or a homepage (if applicable).

**Version of the command being documented (if known):**

### CODE REVIEW

File: pages.zh/linux/openvpn3.md [hunk 1]
```diff
@@ -0,0 +1,23 @@
+# openvpn3
+
+> OpenVPN 3 Linux 客户端。
+> 更多信息： <https://community.openvpn.net/openvpn/wiki/OpenVPN3Linux>.
```
Comment (navarroaxel) [inline pages.zh/linux/openvpn3.md:4]:
```suggestion
> 更多信息：<https://community.openvpn.net/openvpn/wiki/OpenVPN3Linux>.

```

**Review by navarroaxel — COMMENTED** [review]
Hi @wandersofb, welcome and thanks for your submission.

Please sign the license to pass the CLA check.

File: pages.zh/linux/openvpn3.md [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+- 打开一个新的 VPN 会话：
+
+`openvpn3 session-start --config {{path/to/config.conf}}`
```
[the remark this answers is not in the document]
Comment (wandersofb) [reply pages.zh/linux/openvpn3.md:7]: agree

File: pages.zh/linux/openvpn3.md [hunk 3]
```diff
@@ -0,0 +1,24 @@
+# openvpn3
+
+> OpenVPN 3 Linux 客户端。
+> 更多信息：<https://community.openvpn.net/openvpn/wiki/OpenVPN3Linux>.
+
+- 打开一个新的 VPN 会话：
+
+`openvpn3 session-start --config {{路径/config.conf}}`
```
Comment (blueskyson) [inline pages.zh/linux/openvpn3.md:8]:
```suggestion
`openvpn3 session-start --config {{路径/到/config.conf}}`
```

File: pages.zh/linux/openvpn3.md [hunk 4]
```diff
@@ -0,0 +1,24 @@
...
+
+- 列出已建立的会话：
+
+`openvpn3 sessions-list`
+
+- 断开当前建立的以给定配置开始的会话：
+
+`openvpn3 session-manage --config {{路径/config.conf}} --disconnect`
```
Comment (blueskyson) [inline pages.zh/linux/openvpn3.md:16]:
```suggestion
`openvpn3 session-manage --config {{路径/到/config.conf}} --disconnect`
```

File: pages.zh/linux/openvpn3.md [hunk 5]
```diff
@@ -0,0 +1,24 @@
...
+
+- 导入 VPN 配置：
+
+`openvpn3 config-import --config {{路径/config.conf}}`
```
Comment (blueskyson) [inline pages.zh/linux/openvpn3.md:20]:
```suggestion
`openvpn3 config-import --config {{路径/到/config.conf}}`
```

File: pages.zh/linux/openvpn3.md (hunk 3 above)
Comment (wandersofb) [reply pages.zh/linux/openvpn3.md:8]: > 
@blueskyson, Thanks for your comments. You are exactly right, but this kind of translation is a bit redundant, maybe it's easier to remove '到/‘. This is my first PR. Hopefully, you understand what I said.

Comment (wandersofb) [reply pages.zh/linux/openvpn3.md:8]: @blueskyson , you can check [this](https://github.com/tldr-pages/tldr/blob/main/pages.zh/linux/asciiart.md) translation.

Comment (blueskyson) [reply pages.zh/linux/openvpn3.md:8]: We have a [common-argument.md](https://github.com/tldr-pages/tldr/blob/main/contributing-guides/translation-templates/common-arguments.md) for normalizing commonly used terms. We want tldr looks normalized among these terms, so I suggest using `路径/到/...` for all `path/to/some_file`.

Comment (blueskyson) [reply pages.zh/linux/openvpn3.md:8]: `asciiart.md` might created before we creating [common-argument.md](https://github.com/tldr-pages/tldr/blob/main/contributing-guides/translation-templates/common-arguments.md). We may fix `asciiart.md`.

Comment (wandersofb) [reply pages.zh/linux/openvpn3.md:8]: Ok, I will fix that. Thanks for your review and suggestion.

**Comment by wandersofb** [comment]
> Hi @wandersofb, welcome and thanks for your submission.
> 
> Please sign the license to pass the CLA check.

**Review by blueskyson — APPROVED** [review]
Thanks for contribution!

**Review by navarroaxel — APPROVED** [review]

**Pull request merged**